OpenRA.Game/Activities/Activity.cs

```diff
@@ -266,6 +282,11 @@ public virtual IEnumerable<Target> GetTargets(Actor self)
 		{
 			yield break;
 		}
+
+		public virtual TargetLineNode TargetLineNode(Actor self)
```

Comment:
Please keep this as an `IEnumerable<TargetLineNode>`. Composite activities may need to return more than one node.